tests/test_html5lib_shim.py

```diff
@@ -117,6 +117,11 @@ def test_serializer(data, expected):
         {},
         '<a href=\'http://example.com\'\'>',
         '<a href="http://example.com"></a>'
+    ),
+    (
+        {},
+        '</ chars',
+        '&lt;/ chars',
```

Comment:
This test case doesn't have anything to do with "invalid-character-in-attribute-name" testing, so I'd add a comment along these lines:

```
# Test that "expected-closing-tag-but-got-char" works when tags is None
```